core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -309,6 +311,8 @@ oppia.directive('conversationSkin', [
           UserService.getUserInfoAsync().then(function(userInfo) {
             $scope.isLoggedIn = userInfo.isLoggedIn();
           });
+          $templateCache.put('feedback_popup_container.html',
+            '<feedback-popup></feedback-popup>');
```

Comment:
Put `<feedback-popup></feedback-popup>` into angular constant and use the constant here.

Reply:
Done.